packages/contracts/package.json

```diff
@@ -8,12 +8,13 @@
     "hardhat": "^2.7.1"
   },
   "scripts": {
-    "build:forge": "forge build --root . -c contracts --out forge-artifacts",
```

Comment:
have to use this flag because forge doesn't seem to actually recognize the foundry.toml file. I think because it's confused by the dir structure of the repo.